README.md

```diff
@@ -1,297 +1,236 @@
 
-This is a helper class for use with the Data API. It creates the initialisation packet and sends a request to the Data API, returning an instance of Remote. You can then interact as you would with Remote, e.g., `getBody()`
+### <b>Key Learnosity concepts</b>
+Want more general information about how apps on Learnosity actually work? Take a look at our [Key Learnosity Concepts page](https://help.learnosity.com/hc/en-us/articles/360000754638-Key-Learnosity-Concepts).
 
-#### request()
+### <b>Glossary</b>
+Need an explanation for the unique Learnosity meanings for Item, Activity and Item bank? See our [Glossary of Learnosity-specific terms](https://help.learnosity.com/hc/en-us/articles/360000754838-Glossary-of-Learnosity-and-Industry-Terms).
 
-Used for a single request to the Data API. You can call as many times as necessary. It will return a `Remote` object, on which `getBody()` needs to be called to get the contents of the response.
+(Back to top)
 
-``` php
-$DataApi = new DataApi();
-$response = $DataApi->request(
-    'https://data.learnosity.com/v1/itembank/items',
-    [
-       'consumer_key' => 'yis0TYCu7U9V4o7M',
-       'domain'       => 'localhost'
-    ],
-    'superfragilisticexpialidocious',
-    [
-       'limit' => 20
-    ],
-    'get' // optional, will default to 'get' in the backend if unspecified
-);
-```
+##  Production deployment via Composer
 
-#### requestRecursive()
+Using [Composer] is the recommended way to install the Learnosity SDK for PHP in production. The easiest way is to run this from your parent project folder:
 
-Used to make recursive requests to the Data API, using the *next* token, for as much data is returned based on the request filtering provided.
+    composer require learnosity/learnosity-sdk-php "^1.0"
 
-You can pass a callback as the 6th argument, that will be executed upon completion of every request.
+You can also specify the requirement manually by adding "learnosity/learnosity-sdk-php" as a dependency in your project's composer.json file.
 
-``` php
-$DataApi = new DataApi();
-
-$response = $DataApi->requestRecursive(
-    'https://data.learnosity.com/v1/itembank/items',
-    [
-       'consumer_key' => 'yis0TYCu7U9V4o7M',
-       'domain'       => 'localhost'
-    ],
-    'superfragilisticexpialidocious',
-    [
-       'limit' => 20
-    ],
-    'get',
-    function ($data) {
-        $this->processData($data);
+    {
+        "require": {
+            "learnosity/learnosity-sdk-php": "^1.0"
+        }
     }
-);
-
-function processData($data)
-{
-    // Do something with $data
-}
-```
-
-#### Overriding the remote options
-
-You can send an array to the DataAPI constructor to override any remote (cURL) options, e.g.:
-
-``` php
-$options = array(
-    'connect_timeout' => 20
-    'timeout' => 60
-);
 
-$dataapi = new DataApi($options);
-```
+Then, install the new dependency with
 
+    composer update learnosity/learnosity-sdk-php
 
-## Autoload
+### <b>Autoload</b>
```

Comment:
Bold in Markdown is `**bold**`, but I don't think it's needed here.

Reply:
In my opinion, the default H3 heading styles defined in these heading levels is too light. I wanted to bump them up, to make them more distinct and visible.

Reply:
I've adjust it to be Markdown bolding.